x-pack/test/case_api_integration/basic/tests/cases/find_cases.ts

```diff
@@ -6,246 +6,453 @@
  */
 
 import expect from '@kbn/expect';
+import supertestAsPromised from 'supertest-as-promised';
+import { SearchResponse } from 'elasticsearch';
 import { FtrProviderContext } from '../../../common/ftr_provider_context';
 
 import { CASES_URL } from '../../../../../plugins/case/common/constants';
 import { postCaseReq, postCommentUserReq, findCasesResp } from '../../../common/lib/mock';
 import { deleteAllCaseItems } from '../../../common/lib/utils';
+import { CasesFindResponse } from '../../../../../plugins/case/common/api';
+
+interface CaseAttributes {
+  cases: {
+    title: string;
+  };
+}
 
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
   const es = getService('es');
   describe('find_cases', () => {
-    afterEach(async () => {
-      await deleteAllCaseItems(es);
-    });
+    describe('basic tests', () => {
```

Comment:
These changes are because I created a new `describe` block for the already existing tests and for the new pagination tests. These changes are really just bringing the older tests under the new describe block and indentation.